src/dev/jest/config.js

```diff
@@ -101,4 +101,7 @@ export default {
     '<rootDir>/node_modules/enzyme-to-json/serializer',
   ],
   reporters: ['default', '<rootDir>/src/dev/jest/junit_reporter.js'],
+  globals: {
+    Uint8Array: Uint8Array,
+  },
```

Comment:
There are multiple places in the AWS Encryption SDK that dataKey / plainText is buffer but recognized as object instead of uint8array during jest test case running

```
function unwrapDataKey(dataKey) {
    if (dataKey instanceof Uint8Array)
        return dataKey;
    if (exports.supportsKeyObject && dataKey instanceof exports.supportsKeyObject.KeyObject)
        return dataKey.export();
    throw new Error('Unsupported dataKey type');
}
```
found an known issue - https://github.com/facebook/jest/issues/4422

Using this solution helped 

https://github.com/facebook/jest/issues/4422#issuecomment-770274099

Reply:
will it impact other test execution?

Reply:
Don't think so. It's considered a bug in the Jest framework from the issue above. Also, I ran 'yarn test:jest' on local, and it appears that nothing was broken.